beater/cloudbeat.go

```diff
@@ -257,7 +252,10 @@ func (bt *cloudbeat) configUpdate(update *common.Config) error {
 		return fmt.Errorf("could not marshal to YAML: %w", err)
 	}
 
-	if err := csppolicies.HostBundleWithDataYaml("bundle.tar.gz", policies, y); err != nil {
+	bundle := csppolicies.CISKubernetesBundle()
+	bundle.With("data.yaml", y)
```

Comment:
Will this work?
```suggestion
	bundle := csppolicies.CISKubernetesBundle().With("data.yaml", y)
```

Reply:
The method expects a pointer in order to modify
```
cannot call pointer method With on "github.com/elastic/csp-security-policies/bundle".Bundlecompiler[InvalidMethodExpr](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#InvalidMethodExpr)
```
Attempted to ref
```
bundle := (&csppolicies.CISKubernetesBundle()).With("data.yaml", y)
```
And got
```
invalid operation: cannot take address of csppolicies.CISKubernetesBundle() (value of type "github.com/elastic/csp-security-policies/bundle".Bundle)compiler[UnaddressableOperand](https://pkg.go.dev/golang.org/x/tools/internal/typesinternal?utm_source%3Dgopls#UnaddressableOperand)
```
I can refactor the CSP policies code or do workarounds here (which are probably worse than having two lines), but I think that this code will be removed in the near future and there isn't much value in that, what do you think? @eyalkraft

Reply:
yep, let's keep it that way.